test/modules/geometry-calculation.test.js

```diff
@@ -9,108 +9,118 @@ let point4 = new Point({ x: 2, y: 1 })
 let point5 = new Point({ x: 2, y: 2 })
 let point6 = new Point({ x: 1, y: 1 })
 
-
-let line1 = new Line({point1, point2,slope:2,n:3})
-let line2 = new Line({point3, point4,slope:2,n:5})
-let line3 = new Line({point3, point4,slope:4,n:3})
-let line4 = new Line({point5, point6,slope:1,n:0})
-let line5 = new Line({point5, point4,slope:1,n:3})
-let line6 = new Line({point5, point4,slope:1,n:3})
-
-
-
+let line1 = new Line({ point1, point2, slope: 2, n: 3 })
+let line2 = new Line({ point3, point4, slope: 2, n: 5 })
+let line3 = new Line({ point3, point4, slope: 4, n: 3 })
+let line4 = new Line({ point5, point6, slope: 1, n: 0 })
+let line5 = new Line({ point5, point4, slope: 1, n: 3 })
+let line6 = new Line({ point5, point4, slope: 1, n: 3 })
 
```

Comment:
don't declare parameters in the head of the module, each test gets its own object to test on it